tests/appsec/iast/source/test_cookie_name.py

```diff
@@ -16,6 +16,10 @@ class TestCookieName(BaseSourceTest):
     source_name = "user"
     source_value = "user"
 
+    @bug(library="nodejs", reason="not reported correctly")
```

Comment:
Is it a bug? Or just something that we can't do correctly by node limitiations?

Reply:
`cookie_name` and `header_name` tests are not reporting the vulnerability in the tests but I've made several tries to discover the reason without luck.

Everything is working correctly while debugging the test app manually or in our showcase...

Reply:
by now, I've removed both `cookie_name` and `header_name` tests in nodejs manifest.